llvm/tools/llvm-cov/CoverageExporterJson.cpp

```diff
@@ -125,7 +125,7 @@ json::Array renderRegions(ArrayRef<coverage::CountedRegion> Regions) {
 json::Array renderBranchRegions(ArrayRef<coverage::CountedRegion> Regions) {
   json::Array RegionArray;
   for (const auto &Region : Regions)
-    if (!Region.Folded)
+    if (!Region.TrueFolded || !Region.FalseFolded)
```

Comment:
replace with a helper function? e.g. Region.isFolded()

Reply:
I guess we have to revisit how to show folding. Let me feed this to future tasks.